crudApp/src/App.js

```diff
@@ -0,0 +1,26 @@
+//Overview of my CRUD Application:
+//I have added three routes Home Add and Edit which is used to perform respectives task.
+//I have used redux for maintaining the state of Employees state which is basically a list of employees.
+//I have added a button called "GET" which is used to fetch data from an api which I created using website
+//https://mocki.io/fake-json-api where I basically stored a list of employees in json form and fetched it
+//when get button gets clicked.
+
+import Home from "./components/Home";
+import "./styles.css";
```

Comment:
import sequence should be 
- react imports
- any packages related imports
- shared folders which are in the same app
- utils and constants
- styles

Fix all the imports in your project